ouroboros-consensus/src/ouroboros-consensus/Ouroboros/Consensus/Ledger/Tables.hs

```diff
@@ -8,12 +8,26 @@
 {-# LANGUAGE TypeOperators            #-}
 {-# LANGUAGE UndecidableInstances     #-}
 
--- | See @'LedgerTables'@
+-- | This module defines the 'LedgerTables', a portion of the Ledger notion of a
```

Comment:
I think this section could benefit from some concrete examples (with some simple ledger state). Also we could give examples of the different instantiation of ledger tables (eg sets, diff seqs, etc).

Reply:
Done